node/network/approval-distribution/src/lib.rs

```diff
@@ -256,56 +412,120 @@ struct BlockEntry {
 	/// This maps to their knowledge of messages.
 	known_by: HashMap<PeerId, PeerKnowledge>,
 	/// The number of the block.
-	number: BlockNumber,
+	pub number: BlockNumber,
```

Comment:
This is just stylistic, but I generally don't like to see `pub` fields mised with behavior, i.e. `pub` when POD, private otherwise